src/test/scala/sigmastate/lang/SigmaTyperTest.scala

```diff
@@ -80,13 +80,13 @@ class SigmaTyperTest extends PropSpec with PropertyChecks with Matchers with Lan
   property("predefined functions") {
     typecheck(env, "allOf") shouldBe AllSym.tpe
     typecheck(env, "allOf(Array(c1, c2))") shouldBe SBoolean
-    typecheck(env, "getVar[Byte](10)") shouldBe SByte
-    typecheck(env, "getVar[Array[Byte]](10)") shouldBe SByteArray
-    typecheck(env, "getVar[SigmaProp](10)") shouldBe SSigmaProp
-    typecheck(env, "p1 && getVar[SigmaProp](10)") shouldBe SBoolean
-    typecheck(env, "getVar[SigmaProp](10) || p2") shouldBe SBoolean
-    typecheck(env, "getVar[SigmaProp](10) && getVar[SigmaProp](11)") shouldBe SBoolean
-    typecheck(env, "Array(true, getVar[SigmaProp](11))") shouldBe SCollection(SBoolean)
+    typecheck(env, "getVar[Byte](10).get") shouldBe SByte
```

Comment:
Checks for `SOption` types (e.g. `typecheck(env, "getVar[Byte](10)") shouldBe SOption(SByte)`) seems to be missing here.

Reply:
@greenhat Thanks, for catching this up.